sig/datadog/appsec/waf.rbs

```diff
@@ -2,6 +2,8 @@ module Datadog
   module AppSec
     module WAF
       type data = String | Symbol | Integer | Float | TrueClass | FalseClass | Array[data] | Hash[(String | Symbol | nil), data] | nil
+      type known_addresses = ::Array[::String?]
```

Comment:
Just to be sure, are we really might return `String` or `nil` here? Maybe it's a leftover?

Reply:
yes, sorry, will fix it

Reply:
`known_addresses` uses `get_array_of_strings` from ffi gem, that defines the return type as `::Array[::String]?`:
https://github.com/ffi/ffi/blob/5b44581847bf167b83db51ac64aa409ccc9cabee/sig/ffi/abstract_memory.rbs#L110